.github/workflows/add-preview-domain.yml

```diff
@@ -0,0 +1,57 @@
+name: Assign preview-domain
+on:
+  # Trigger when a pull request is opened or reopened
+  pull_request:
+    types: [opened, reopened]
+
+permissions:
+  contents: read
+
+jobs:
+  assign-preview-domain:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Prepare domain
+        id: prepare
+        uses: actions/github-script@v7.0.1
+        with:
+          script: |
+            const domain = `${{ github.head_ref }}.${{ vars.PREVIEW_DOMAIN }}`.toLowerCase()
+            core.setOutput('domain', domain)
+
+      - name: Comment with the assigned preview-domain
+        uses: actions/github-script@v7.0.1
+        with:
+          script: |
+            const knownString = '### Preview domain'
+            const pullRequest = await github.rest.pulls.get({
+              pull_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            })
+            const hasAlreadyCommented = pullRequest.data.body.includes(knownString)
+
+            if (hasAlreadyCommented) {
+              console.log('Already commented once')
```

Comment:
Do the consoles add anything?

Reply:
Just for debugging mostly
https://github.com/dailydotdev/apps/actions/runs/8392439290/job/22985086956#step:3:34